emeis/core/models.py

```diff
@@ -168,20 +169,21 @@ class Scope(MPTTModel, UUIDModel):
     description = LocalizedTextField(
         _("scope description"), null=True, blank=True, required=False
     )
-    parent = TreeForeignKey(
-        "self",
-        on_delete=models.CASCADE,
-        null=True,
-        blank=True,
-        related_name="children",
-    )
     is_active = models.BooleanField(default=True)
 
+    objects = TreeQuerySet.as_manager(with_tree_fields=True)
+
+    def save(self, *args, **kwargs):
+        # django-tree-queries does validation in TreeNode.clean(), which is not
+        # called by DRF (only by django forms), so we have to do this here
+        self.clean()
+        return super().save(*args, **kwargs)
+
     def __str__(self):
         return f"{type(self).__name__} ({self.full_name}, pk={self.pk})"
 
     class Meta:
-        ordering = ["full_name"]
+        ordering = ["name"]
```

Comment:
Why this change?

Reply:
Ah, because it's not supported anymore, right?

Reply:
Yes and no. django-tree-queries uses it to sort siblings, and it doesn't support `.order_by()` anymore (which may be problematic in some specific use cases, but I think our frontend doesn't care / expects the output to be in tree-order anyway)